app/screens/content-view-screen/content-view-screen.tsx

```diff
@@ -60,6 +60,12 @@ const injectedJavaScript = `${
   }window.addEventListener('message', function(event) {
     window.ReactNativeWebView.postMessage(JSON.stringify(event.data));
   });
+  ${/* Hide LikeCoin button on custom site */ ""
```

Comment:
How about liker.social? (just curious if we want to handle it, or have a list of app config of custom site for them)